backend/services/tiktok_service.py

```
@@ -127,88 +131,63 @@ async def _test_client_credentials(self, client_key: str, client_secret: str) ->
                             "error": "TikTok API rate limit exceeded. Try again later."
                         }
                     else:
-                        error_text = await response.text()
+                        try:
+                            error_data = await response.json()
+                            error_msg = error_data.get("error", {}).get("message", "Unknown error")
+                        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)